### DISCUSSION TITLE
Looping bug which hard to stop!

### DISCUSSION THREAD

**Discussion by NDruce** [community/community#67940] [Mobile]
This is video: [Dropbox](https://www.dropbox.com/scl/fi/4txg3dr9wq8jwb5q9mcma/7073185737242078758.mp4?rlkey=7q7uj5c2h289allqefwnrta65&dl=0)

As I remembered, I was clicked on changelog link and this bug was started ^

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.128.0 (108490092)
Device: iPhone15,2 (iOS 17.0)
</details>

**Comment by ipetersenpai** [comment]
If you are using mobile emulator try to close the program and open it again.

**Comment by NDruce** [comment]
> If you are using mobile emulator try to close the program and open it again.

No. I using original GitHub app for iPhone without anything in middle. 
I just want to submit bug report to GitHub officials
